### ISSUE TITLE
Bug: Special Characters seem to be mucking up metadata matches...

### ISSUE THREAD

**Issue by Ronald-Diemicke** [JMBeresford/retrom#213]
### Describe the bug
I've got games in my library that have strange symbols like ™ - The metadata search should ignore these.

### Steps To Reproduce
1. Create a folder called 'PRINCE OF PERSIA® THE SANDS OF TIME' or search for this in a manual metadata search
2. Notice how nothing shows up.
3. Then create a folder for 'PRINCE OF PERSIA THE SANDS OF TIME' or search for this in the manual metadata search
4. notice how the special character restricts it from being found.

### Expected behavior
Maybe it should try to match w/ special characters first, then strip them and try again? It should try to match at all costs.

### Environment (please complete the following information _where applicable_):
 - OS: Running web client on Unraid Docker Container
 - Browser: Chrome
 - Version: 129.0.6668.91

**Comment by JMBeresford** [comment]
Agreed, stripping these characters ( or otherwise delineating them ) in searches will be explored soon. Thanks for the report!
